### ISSUE TITLE
3 process errored

### ISSUE THREAD

**Issue by DevDorrejo** [mozilla/fxa#1599]
hello, i have finish installed fxa and when i do status, appear the next process errored
![errored](https://user-images.githubusercontent.com/7697566/59978131-1d96a600-95a7-11e9-8957-a3877f80735b.png)

> ./pm2 log 20 21 22
![log 20](https://user-images.githubusercontent.com/7697566/59978190-d0ff9a80-95a7-11e9-91f3-5a71076757d7.png)
![log 21](https://user-images.githubusercontent.com/7697566/59978191-d2c95e00-95a7-11e9-859c-8945acb86d1c.png)
![log 22](https://user-images.githubusercontent.com/7697566/59978192-d3fa8b00-95a7-11e9-8bce-c51592943586.png)

i tried restart the process, re-download and re-install, use a new user, reboot the pc.

**Comment by chenba** [comment]
@DevDorrejo I was not able to reproduce this today (@ rev `19692d6`).  Can you try it again with the latest `master`?

**Comment by DevDorrejo** [comment]
> @DevDorrejo I was not able to reproduce this today (@ rev `19692d6`). Can you try it again with the latest `master`?

sorry for late response, my laptop died and i am waiting the replacement, the moment i get the new i will test again and update the post.

**Comment by DevDorrejo** [comment]
any new?

**Comment by chenba** [comment]
@DevDorrejo Hi.  At least a couple of people have set up the project recently without running into those errors that you encountered.  

Which version of nodejs are you using?

**Comment by DevDorrejo** [comment]
> @DevDorrejo Hi. At least a couple of people have set up the project recently without running into those errors that you encountered.
> 
> Which version of nodejs are you using?

sorry for late response, i am using npm10

**Comment by chenba** [comment]
@DevDorrejo No one has ran into any issues with node 10 as far as I know.

Could you give more information on your set up?  What OS are you using?  What command did you use to install nodejs?  Anything you think is relevant.  I'll try to reproduce if it's possible.

**Comment by DevDorrejo** [comment]
> @DevDorrejo No one has ran into any issues with node 10 as far as I know.
> 
> Could you give more information on your set up? What OS are you using? What command did you use to install nodejs? Anything you think is relevant. I'll try to reproduce if it's possible.

I am using OpenSUSE Leap 15.1, and install nodejs from the repo and simple install all the dependencies:

zypper -v in --details <package> or yast2 "software management" for installation of dependencies:
- build-essential > devel_basis pattern (zypper -v in --details --type pattern devel_basis) 
- graphicsmagick > GraphicsMagick
- python-virtualenv > python-virtualenv
- python-dev > python-devel
- libssl-dev > libopenssl-devel
- curl > curl
- openjdk-11-jre > java-11-openjdk & java-11-openjdk-devel (jdk)
- firefox > MozillaFirefox
- pkg-config > pkg-config

**Comment by DevDorrejo** [comment]
/home/wedc/Documents/workspace/fxa/packages/fxa-oauth-console/node_modules/node-sass/build:
![image](https://user-images.githubusercontent.com/7697566/61472980-7dd4f980-a953-11e9-8193-6b129157f6a6.png)
![image](https://user-images.githubusercontent.com/7697566/61473070-ac52d480-a953-11e9-9685-e2cb03361a3e.png)
![image](https://user-images.githubusercontent.com/7697566/61473202-f471f700-a953-11e9-8c05-9bb5564977ca.png)

Continuing the previous image: ![image](https://user-images.githubusercontent.com/7697566/61473369-49ae0880-a954-11e9-82c9-986ba833b3dc.png)

**Comment by philbooth** [comment]
> I am using OpenSUSE Leap 15.1...

Fwiw, I vaguely recall somone reporting problems running on SUSE before, although I didn't manage to find an issue about it in any of our repos.

The problem is that none of us here are using it though, so it's going to be difficult to offer support. Are you able to try a different OS?

**Comment by DevDorrejo** [comment]
no, i can't because i am installing this service on a server base on opensuse leap 15.1.

**Comment by DevDorrejo** [comment]
/home/wedc/Documents/workspace/fxa/packages/support-panel/node_modules/fxa-auth-db-mysql/node_modules/scrypt-hash/build
![image](https://user-images.githubusercontent.com/7697566/61481305-f8a71000-a965-11e9-97d4-8163a39e90d0.png)
![image](https://user-images.githubusercontent.com/7697566/61481614-a6b2ba00-a966-11e9-914c-f7a66665b0ca.png)

**Comment by DevDorrejo** [comment]
the installation have to be made from any user or from root?

**Comment by philbooth** [comment]
You definitely don't need/want to be root.

**Comment by DevDorrejo** [comment]
> You definitely don't need/want to be root.

ok, i had made all installation using a normal use with the respective groups added

**Comment by DevDorrejo** [comment]
something new?

**Comment by shane-tomlinson** [comment]
@DevDorrejo - Unfortunately not. None of us run openSUSE or are well enough familiar with its peculiarities to be able to provide support on that OS, we all run CentOS and Ubuntu.

If any other openSUSE users come by this and have resolved these issues already, help would be appreciated!

**Comment by DevDorrejo** [comment]
Question: 

1- Can the difference between java 8 and java-11-openjdk, create a issue?

**Comment by chenba** [comment]
@DevDorrejo Java is used for the functional/selenium tests.  You shouldn't run into issues without attempting to run those tests.

**Comment by DevDorrejo** [comment]
ok, i am trying debugging and looking for a solution on suse

> @DevDorrejo Java is used for the functional/selenium tests. You shouldn't run into issues without attempting to run those tests.

**Comment by DevDorrejo** [comment]
well ppl, is like i make it work, now i have this (Warning or error?): 
![image](https://user-images.githubusercontent.com/7697566/61910643-33c7b700-af03-11e9-920f-f76f2a8c1873.png)

and after i do ./pm2 status, got not error, still on openSUSE leap 15.1

**Comment by DevDorrejo** [comment]
It's Alive!!!!: https://youtu.be/JZjcdYnt8is

**Comment by chenba** [comment]
🎉 👏 🍻

**Comment by chenba** [comment]
@DevDorrejo Is this issue ready to be closed?

**Comment by DevDorrejo** [comment]
yeah, only answer me this 

> well ppl, is like i make it work, now i have this (Warning or error?):
> ![image](https://user-images.githubusercontent.com/7697566/61910643-33c7b700-af03-11e9-920f-f76f2a8c1873.png)
> 
> and after i do ./pm2 status, got not error, still on openSUSE leap 15.1.
> 
> Yuhuuu!

**Comment by DevDorrejo** [comment]
**SOLUTION:** use nvm for nodejs instead nodejs from the repo, the real problem is that only install success if you use npm6 and not any greater version, it fail with npm8

**Comment by DevDorrejo** [comment]
> well ppl, is like i make it work, now i have this (Warning or error?):
> ![image](https://user-images.githubusercontent.com/7697566/61910643-33c7b700-af03-11e9-920f-f76f2a8c1873.png)
> 
> and after i do ./pm2 status, got not error, still on openSUSE leap 15.1.
> 
> Yuhuuu!

need info about this to finally close the issue.

**Comment by chenba** [comment]
@DevDorrejo What's `./pm2 status` like when you get that error?  What else is in `event-broker-error.log`?  (I haven't seen that error before or have done any work on the event broker, so I don't have a guess.)

**Comment by DevDorrejo** [comment]
maybe because i am using NetworkManager? better use wicked?
